service/src/models/archaeologist.ts

```diff
@@ -168,25 +174,60 @@ export class Archaeologist {
               ) {
                 error = SarcophagusValidationError.INVALID_ARWEAVE_SHARD;
               }
+
+              // Emit error if set for any reason. (Offload burden of user-friendly messaging to recipient)
               if (error) {
-                // offload the burden of user-friendly messaging to the recipient
                 emitError({ code: error, message: "Declined to sign" });
                 return;
               }
+
               // sign sarcophagus parameters to demonstrate agreement
-              const msg = ethers.utils.solidityPack(
+              console.log('diggingFee', diggingFee);
+              console.log('ethers.utils.parseEther(diggingFee).toString()', ethers.utils.parseEther(diggingFee).toString());
+              console.log([
+                arweaveTxId,
+                unencryptedShardDoubleHash,
+                maximumRewrapIntervalBN.toString(),
+                ethers.utils.parseEther(diggingFee).toString(),
+                timestamp.toString(),
+              ]);
+
+              const signPacked = async (
+                types: string[],
+                data: string[],
+              ) => {
+                const dataHex = ethers.utils.defaultAbiCoder.encode(types, data);
+                const dataHash = ethers.utils.keccak256(dataHex);
+                const dataHashBytes = ethers.utils.arrayify(dataHash);
+                const signature = await this.web3Interface.ethWallet.signMessage(dataHashBytes);
+                return signature;
+              }
+
+
+
+              // const msg = ethers.utils.solidityPack(
+              //   ["string", "bytes32", "uint256", "uint256", "uint256"],
+              //   [
+              //     arweaveTxId,
+              //     unencryptedShardDoubleHash,
+              //     maximumRewrapIntervalBN.toString(),
+              //     ethers.utils.parseEther(diggingFee).toString(),
+              //     timestamp.toString(),
+              //   ]
+              // );
+              const signature = await signPacked(
                 ["string", "bytes32", "uint256", "uint256", "uint256"],
                 [
                   arweaveTxId,
                   unencryptedShardDoubleHash,
-                  maxRewrapInterval.toString(),
-                  diggingFee,
+                  maximumRewrapIntervalBN.toString(),
+                  ethers.utils.parseEther(diggingFee).toString(),
```

Comment:
Thinking the digging fee should be parsed raw here (without `parseEther`), and expect that the webapp would send over the full WEI amount of digging fees.